### PR TITLE
Revert change to multiprocessing mode

### PR DESCRIPTION
**Pull request by ashleysommer**
Revert change to multiprocessing mode accidentally included in https://github.com/huge-success/sanic/pull/1853

### CODE REVIEW

File: sanic/server.py [hunk 1]
```diff
@@ -927,7 +927,7 @@ def sig_handler(signal, frame):
 
     signal_func(SIGINT, lambda s, f: sig_handler(s, f))
     signal_func(SIGTERM, lambda s, f: sig_handler(s, f))
-    mp = multiprocessing.get_context("spawn")
+    mp = multiprocessing.get_context("fork")
```
Comment (Tronic) [inline sanic/server.py:930]: This should restore prior functionality. Do note that it won't work on Windows but Sanic doesn't support multiple workers on Windows at this time anyway (although that really ought to be fixed, PRs welcome!).

**Comment by ahopkins** [comment]
Should we include this in `20.6`?

**Comment by Tronic** [comment]
> Should we include this in `20.6`?

Yes, absolutely, and I believe that with 20.6.1 people will get major b0rkage in their apps because not even the standard Hello World will work with multiple workers. Perhaps roll another release today with this and the unix socket PR? The pickling issue is perhaps important enough to add specific tests to avoid it happening again (actually I am surprised that the current multiple worker tests do not hit it at all), but then that is probably better left for a later release.

Great work with getting the release rolled out despite all the difficulties. In future hopefully we can get PRs merged in good time prior to release, to have releases rolled out in a more orderly fashion.

**Comment by ahopkins** [comment]
Github is throwing 500's right now on merges. Will try again in a little bit to see if they resolved.

**Comment by ahopkins** [comment]
This pull request has been mentioned on **Sanic Community Discussion**. There might be relevant details there:

https://community.sanicframework.org/t/multiprocessing-is-currently-not-supported-on-nt/827/2

**Pull request merged**